templates/default.yaml

```diff
@@ -256,6 +256,8 @@ containerd:
 # # It requires `ansible-playbook` command to be installed.
 # # Environment variables such as ANSIBLE_CONFIG can be used, to control the behavior of the playbook execution.
 # # See ansible docs, and `ansible-config`, for more info https://docs.ansible.com/ansible/latest/playbook_guide/
+# # DEPRECATED The ansible mode is deprecated, and should not be used. Instead call ansible-playbook directly,
+# # either from the host after the instance is started or from the instance by running ansible locally instead.
```

Comment:
You can add an `/examples/ansible` page under https://lima-vm.io/docs/examples/ for ansible-playbook